package.json

```diff
@@ -49,7 +49,7 @@
     "dependencies": {
         "@azure-tools/autorest-extension-base": "^3.1.235",
         "@azure-tools/codegen": "^2.4.257",
```

Comment:
how about this one? and please bump the modelerfour version per the new python codegen version. thx

Reply:
Bumped both modelerfour and codegen. Thanks!